inc/ThirdParty/Plugins/PageBuilder/Elementor.php

```diff
@@ -166,4 +168,33 @@ public function exclude_js( $excluded_files ): array {
 
 		return $excluded_files;
 	}
+
+	/**
+	 * Remove rocket metabox option from post.
+	 *
+	 * @param array $cpts Custom post type.
+	 * @return array
+	 */
+	public function remove_rocket_metabox_option( array $cpts ): array {
+		if ( isset( $cpts['elementor_library'] ) ) {
+			unset( $cpts['elementor_library'] );
+
+			return $cpts;
+		}
```

Comment:
```suggestion
		if ( isset( $cpts['elementor_library'] ) ) {
			unset( $cpts['elementor_library'] );
                }

		return $cpts;
```

We want to always return something